app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = list()
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (
+            "{"
+            f"Name: {self.name}, "
+            f"Health: {self.health}, "
+            f"Hidden: {self.hidden}"
+            "}"
+        )
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        self.hidden = True if not self.hidden else False
```

Comment:
Reverse the value of `self.hidden` without using `if else` but with the use of `not` as mentioned in the [checklist](https://github.com/Quiet-Klirik/py-herbivores-and-carnivores/blob/solution/yaroslav-lysiuk/checklist.md)